_studio/shared/umc/codec/h265_dec/src/umc_h265_mfx_supplier.cpp

```diff
@@ -252,10 +252,18 @@ void MFXTaskSupplier_H265::SetVideoParams(mfxVideoParam * par)
 
 UMC::Status MFXTaskSupplier_H265::FillVideoParam(mfxVideoParam *par, bool full)
 {
+    const H265VideoParamSet * vps = GetHeaders()->m_VideoParams.GetCurrentHeader();
     const H265SeqParamSet * seq = GetHeaders()->m_SeqParams.GetCurrentHeader();
+
     if (!seq)
         return UMC::UMC_ERR_FAILED;
 
+    if (vps != nullptr && (vps->getTimingInfo()->vps_timing_info_present_flag || full))
+    {
+        par->mfx.FrameInfo.FrameRateExtD = vps->getTimingInfo()->vps_num_units_in_tick;
+        par->mfx.FrameInfo.FrameRateExtN = vps->getTimingInfo()->vps_time_scale;
+    }
+
```

Comment:
This contradicts the rest code. I mean we fill all params inside FillVideoParam() and only FrameRate here. Any special reason for it?

Reply:
The FrameRate can get from VPS or SPS, but FillVideoParam() only gets SPS as a parameter. So add FrameRate set based on VPS here.

Reply:
> The FrameRate can get from VPS or SPS, but FillVideoParam() only gets SPS as a parameter. 

Why not to add VPS as well?

Reply:
Add VPS into FillVideoParam() done.